.github/workflows/lmi-dist-deps-build.yml

```diff
@@ -3,6 +3,10 @@ name: LMI-Dist dependency build
 on:
   workflow_dispatch:
 
+permissions:
```

Comment:
What this permission is used for?

Reply:
permission is used for grant access to secuity tokens: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
Why we don't need this before?